flocx_market/cmd/app.py

```diff
@@ -0,0 +1,11 @@
+from flocx_market.api import app
+from flocx_market.db.orm import orm
+from flask_migrate import Migrate
+
+application = app.create_app(app_name='flocx-market')
+migrate = Migrate(application, orm)
+
+if __name__ == '__main__':
+
+    orm.init_app(application)
+    application.run(port=8080, debug=True)
```

Comment:
I'm not sure why we want this over cmd/api.py. I'd keep cmd/api.py for starting up the api, and make any changes needed there.

Reply:
This is the only thing I have for starting up the api. SQLalchemy needs to be initialized when the app/api starts and application.run is starting up the api

Reply:
Either way, we should be updated cmd/api.py instead.

Reply:
I'm a bit unclear on what are you referring to with updating cmd/api.py. Should I just rename cmd/app.py to api.py?

Reply:
Oh, so there's merged code from @DanNiESh that already starts up a flask app, using the code in flocx_market/cmd/api.py. We should continue to use that!

Reply:
> I'm a bit unclear on what are you referring to with updating cmd/api.py. Should I just rename cmd/app.py to api.py?

Please fetch the upstream, and use cmd: flocx-market-api to start the project. This cmd will run cmd/api.py.